docs/api.md

```diff
@@ -149,3 +149,18 @@ hardware does not match a known profile, the value "unknown" is used.
   host or there is a problem with the host itself. Refer to the
   *errorMessage* field in the status section for more details about
   the error condition.
+
+## Triggering Provisioning
+
+Several conditions must be met in order to initiate provisioning.
+
+1. The host spec must have an image with a URL pointing to a valid
+   image file visible from within the cluster and from the host
+   receiving the image.
+2. The host provisioning image URL field must be empty, indicating
+   that no other image has been provisioned to the host. To reuse an
+   existing host with a different image, deprovision the host first.
+3. The host must be marked online so that the operator will keep the
```

Comment:
nit: I suggest being more specific about the API: "The host spec must have `Online` set to `true` so that the controller knows to keep it powered on."

Reply:
Done